src/pages/Compare/index.spec.tsx

```diff
@@ -490,6 +491,61 @@ describe(__filename, () => {
     });
   });
 
+  it('does not dispatch fetchVersionFile() if the selected file was deleted', () => {
+    const addonId = nextUniqueId();
+    const baseVersionId = nextUniqueId();
+    const headVersionId = baseVersionId + 1;
+    const path = 'path.js';
+
+    const version = createExternalVersionWithEntries([{ path, status: 'D' }], {
+      selected_file: path,
+      id: headVersionId,
+    });
+    const store = configureStore();
+
+    dispatchLoadVersionInfo({ store, version });
+
+    store.dispatch(
+      fileTreeActions.buildTree({
+        version: createInternalVersion(version),
+        comparedToVersionId: baseVersionId,
+      }),
+    );
+
+    store.dispatch(
+      versionsActions.loadEntryStatusMap({
+        version,
+        comparedToVersionId: baseVersionId,
+      }),
+    );
+
+    store.dispatch(
+      versionsActions.loadDiff({
+        addonId,
+        baseVersionId,
+        headVersionId,
+        version,
+      }),
+    );
+
+    store.dispatch(
+      versionsActions.setCurrentVersionId({ versionId: version.id }),
+    );
```

Comment:
This seems like a lot of set up for a pretty simple test. I know it's needed, but we also seem to do it in some other tests in this file, which makes me think it would be a good candidate for a helper function. Maybe each test is a bit unique in terms of what it's passing into each of these dispatches, but I suspect at least some of them are exactly the same.

It's not really something to block this PR on, but ideally I'd like to see this consolidated into a helper.

Reply:
Filed #1413